Example/Tests/FriendTests/FriendListTests/Mocks/MockFriendListViewModel.swift

```diff
@@ -13,11 +13,13 @@ class MockFriendListViewModel: FriendListViewModelType {
   
   var isFriendListFound: Bool
   var isFriendListNotFound: Bool
+  var isProfileImageFound: Bool
   
   init() {
     
     isFriendListFound = false
     isFriendListNotFound = false
+    isProfileImageFound = false
```

Comment:
Hmm, the init simply sets these 3 items to false. Why not set their initial value as false directly and not need an init.